systemtest/src/main/java/io/strimzi/systemtest/utils/StUtils.java

```diff
@@ -132,4 +138,40 @@ public static void checkCOlogForUsedVariable(String varName) {
         assertThat(coLog.contains("User defined container template environment variable " + varName + " is already in use and will be ignored"), is(true));
         LOGGER.info("ClusterOperator logs contains proper warning");
     }
+
+    /**
+     * Translate key/value pairs formatted like properties into a Map
+     * @param keyValuePairs Pairs in key=value format; pairs are separated by newlines
+     * @return THe map of key/values
+     */
+    @SuppressWarnings("unchecked")
+    public static Map<String, Object> loadProperties(String keyValuePairs) {
+        try {
+            Properties actual = new Properties();
+            actual.load(new StringReader(keyValuePairs));
+            return (Map) actual;
+        } catch (IOException e) {
+            throw new AssertionError("Invalid Properties definiton", e);
+        }
+    }
+
+    /**
+     * Get a Map of properties from an environment variable in json.
+     * @param json The json from which to extract properties
+     * @param envVar The environment variable name
+     * @return The properties which the variable contains
+     */
+    public static Map<String, Object> getPropertiesFromJson(String json, String envVar) {
```

Comment:
This is a bit bizarre because it returns the env vars for all containers in the (I'm guessing) Pod. If there are duplicates then I _guess_ there's a last-in-wins policy. I suspect this is not really the contract that you want, and that you should therefore make the caller pass a container name or index to get the environment for a particular container (which makes much more conceptual sense).